types/googlemaps/index.d.ts

```diff
@@ -3160,7 +3160,7 @@ declare namespace google.maps {
         }
 
         interface OpeningHours {
-            open_now: boolean;
+            isOpen(): boolean;
```

Comment:
Please use the signature from the [documentation].

```suggestion
            isOpen(date?: Date): boolean;
```

[documentation]: https://developers.google.com/maps/documentation/javascript/reference/places-service#PlaceOpeningHours.isOpen